src/algebra/group/units.lean

```diff
@@ -245,9 +245,17 @@ theorem is_unit_one [monoid M] : is_unit (1:M) := ⟨1, rfl⟩
   (a b : M) (h : a * b = 1) : is_unit a :=
 ⟨units.mk_of_mul_eq_one a b h, rfl⟩
 
+@[to_additive is_add_unit.exists_neg] theorem is_unit.exists_right_inv [monoid M]
+  {a : M} (h : is_unit a) : ∃ b, a * b = 1 :=
+by { rcases h with ⟨⟨a, b, hab, _⟩, rfl⟩, exact ⟨b, hab⟩ }
+
+@[to_additive is_add_unit.exists_neg] theorem is_unit.exists_left_inv [monoid M]
+  {a : M} (h : is_unit a) : ∃ b, b * a = 1 :=
+by { rcases h with ⟨⟨a, b, _, hba⟩, rfl⟩, exact ⟨b, hba⟩ }
```

Comment:
You're using the same target name for both of these. Is the a reason we have to tell to_additive how to name all these lemmas?

Reply:
```suggestion
@[to_additive is_add_unit.exists_neg] theorem is_unit.exists_right_inv [monoid M]
  {a : M} (h : is_unit a) : ∃ b, a * b = 1 :=
by { rcases h with ⟨⟨a, b, hab, _⟩, rfl⟩, exact ⟨b, hab⟩ }

@[to_additive is_add_unit.exists_neg'] theorem is_unit.exists_left_inv [monoid M]
  {a : M} (h : is_unit a) : ∃ b, b * a = 1 :=
by { rcases h with ⟨⟨a, b, _, hba⟩, rfl⟩, exact ⟨b, hba⟩ }
```

Reply:
I think it's just because we don't tend to use `left_neg` and `right_neg`, and someone decided previously they wanted to use primes instead for this.